core/build.gradle

```diff
@@ -30,6 +30,8 @@ dependencies {
 
     // Generic model support
     compile("ai.h2o:h2o-genmodel:${h2oVersion}")
+    // H2O security
+    compile("ai.h2o:h2o-security:${h2oVersion}")
```

Comment:
This should be added as part of different PR after upgrade to new H2O :)

Reply:
Yup, no problem. I just wanted to make it running against the current version of the `rel-yau` branch.

Reply:
Yup

Reply:
@mn-mikke since this is an issue for you right now - should there be a dependency in h2o-core on h2o-security even though it doesn't actually need that

I want to avoid issues not just for SW but for other people building H2O as well

Reply:
@michalkurka was thinking about this, on Sparkling Water side we simply fix it, but as you mentioned, it can affect other folks. I think it's actually a good idea to mark it as a dependency for maven purposes

Reply:
But it's also a little bit hack to create such artificial dependency, not sure

Reply:
IMHO, there are very few customers like us referencing individual h2o projects. The vast majority will assembly jar. Maybe, it would be safer to such a change in a major release.